packages/beacon-node/src/chain/bls/interface.ts

```diff
@@ -22,6 +22,14 @@ export type VerifySignatureOpts = {
   priority?: boolean;
 };
 
+export interface QueuedVerificationOptions extends VerifySignatureOpts {
```

Comment:
Nitpick but weird that one is spelled `Options` while the other one is `Opts`.

Reply:
Nit noted and its a good point!!  I actually thought about this as well but love to fully spell words and didn't want to grow the scope changes.  Updated!